projects/fsastorefrontstyles/scss/components/account/_premium-calendar.scss

```diff
@@ -1,27 +1,136 @@
+      grid-template-columns: repeat(2, 1fr);
+      grid-gap: 1.25rem;
+    }
+    @include media-breakpoint-down(sm) {
+      grid-template-columns: 100%;
+    }
+    .row {
+      @include media-breakpoint-down(md) {
+        display: grid;
+        place-items: center;
+        text-align: center;
+        margin: 0;
+      }
+    }
+    .border:nth-last-of-type(1) {
+      @include media-breakpoint-down(md) {
+        padding: 1.25rem 0.94rem;
+        height: 5.875rem;
+        font-size: var(--cx-h6-font-size);
+        color: var(--cx-color-6);
+      }
+    }
+  }
   .premium-data-row {
     background-color: var(--cx-color-4);
     border-top: none;
-    padding: 26px 0;
+    padding: 1.625rem 0;
+    @include media-breakpoint-down(md) {
+      padding: 0;
+      grid-template-columns: repeat(2, 50%);
+      grid-template-rows: 50% 5.875rem 5.875rem;
+      background-color: var(--cx-color-1);
+      border-radius: 0.625rem 0.625rem 0 0;
+    }
+    .col:first-child,
+    .col:nth-child(2) {
+      @include media-breakpoint-down(md) {
+        padding: 1.25rem 0.94rem;
+        grid-column: 1 / 3;
+        font-size: var(--cx-h6-font-size);
+        color: var(--cx-color-6);
+      }
+    }
+    .col:first-child {
+      @include media-breakpoint-down(md) {
+        @include linear-gradient-background(
+          var(--cx-color-7),
+          var(--cx-color-primary),
+          30%,
+          100%
+        );
+        align-self: stretch;
+        border-radius: 0.625rem 0.625rem 0 0;
+        background-color: var(--cx-color-7);
+      }
+      .col-4 {
+        @include media-breakpoint-down(md) {
+          padding-bottom: 1.25rem;
+          display: grid;
+          justify-items: center;
+        }
+      }
+      .col-8 {
+        @include media-breakpoint-down(md) {
+          font-size: var(--cx-h5-font-size);
+          color: var(--cx-color-1);
+        }
+      }
+    }
+  }
+  .notice {
+    @include media-breakpoint-down(md) {
+      position: unset;
+      display: grid;
+      align-self: stretch;
+      font-size: var(--cx-h6-font-size);
+      font-weight: $font-weight-semi;
+      background-color: var(--cx-color-4);
+    }
+    h5 {
+      @include media-breakpoint-down(md) {
+        position: relative;
+        align-self: end;
+        color: var(--cx-color-6);
+      }
+    }
+    .icon-triangle-bottom {
+      @include media-breakpoint-down(md) {
+        right: 0;
+        top: 50%;
+        @include transition;
+      }
+      &.rotate {
+        transform: rotateZ(180deg);
+        -webkit-transform: rotateZ(180deg);
+      }
+    }
   }
   .accordion {
     max-height: 0px;
     overflow-y: hidden;
     transition: ease-in-out 400ms max-height;
     &.opened {
-      max-height: 600px;
+      max-height: 37.5rem;
       transition: ease-in-out 600ms max-height;
     }
     .container-fluid {
       border: 1px solid var(--cx-color-3);
       border-top: none;
       border-bottom: none;
```

Comment:
I know this isn't your code but perhaps we can use BS classes in HTML for this? Please check.